compiler/circle-partitioner/src/PartitionRead.cpp

```diff
@@ -69,13 +92,33 @@ luci::PartitionTable parse_table(const crew::Sections &sections)
       for (auto &item : items)
       {
         if (item.first == _key_underscore)
```

Comment:
When the item.first have just "_" string?

Reply:
Maybe DRAFT code https://github.com/Samsung/ONE/pull/6231/files#diff-8d11f27aebeac12e3f55cb8df554d505279af923c78791b7b3132415590f7625 will help?